clang/test/Driver/sycl-int-footer.cpp

```diff
@@ -25,9 +25,9 @@
 
 /// Check that integration footer can be disabled
 // RUN:  %clangxx -fsycl -fno-sycl-use-footer %s -### 2>&1 \
-// RUN:   | FileCheck -check-prefix NO-FOOTER --implicit-check-not "-fsycl-int-footer" %s
+// RUN:   | FileCheck -check-prefix NO-FOOTER --implicit-check-not "-fsycl-int-footer" --implicit-check-not "-fsycl-use-main-file-name" %s
```

Comment:
Could you also add a test to be sure that `-fsycl-use-main-file-name` isn't used when `-fsycl` isn't used?

Reply:
Added in a54b24a9bdcac9a977afc73d00540d0657c4255f